src/components/register/RegisterForm.tsx

```diff
@@ -44,24 +45,30 @@ const RegisterForm = () => {
   };
 
   return (
-    <RowGroup>
-      <Input
-        id="connect"
-        className="rounded-r-none bg-light-primary-100 focus:bg-light-primary-100"
-        placeholder="Username"
-        value={name}
-        setValue={setName}
-      />
+    <div className="flex flex-col w-full gap-1">
+      <RowGroup>
+        <Input
+          id="connect"
+          className="rounded-r-none bg-light-primary-100 focus:bg-light-primary-100"
+          placeholder="Username"
+          value={name}
+          setValue={setName}
+        />
 
-      <Button
-        className="rounded-l-none"
-        onClick={mutate}
-        loading={isLoading}
-        disabled={!isValid(name)}
-      >
-        {buttonText()}
-      </Button>
-    </RowGroup>
+        <Button
+          className="rounded-l-none"
+          onClick={mutate}
+          loading={isLoading}
+          disabled={!isValid(name)}
+        >
+          {buttonText()}
+        </Button>
+      </RowGroup>
+
+      <ErrorCard show={!isValidConditions.lteTwenty(name)}>
+        Maximum length is 20 chars
```

Comment:
What about i18n?

Reply:
i18n is a server side service

Reply:
and that's not the point of the pr

Reply:
i18n can and must be implemented on the client side, especially when displaying messages before querying anything to the api (how can it be a server side feature in this context ?) 
As such, it should be planned in advance in the front end app to limit future modifications.

It was just a quick note, and indeed it is not the goal of the PR.